plone/recipe/codeanalysis/entry_points.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from tempfile import mkdtemp
+
+import os
+import subprocess
+import sys
+
+
+def check(data):
+    """Run p.r.codeanalysis when releasing with zest.releaser."""
+    from zest.releaser.utils import ask
+    if not ask('Run code analysis before tagging?'):
+        return
+
+    filename = '{0}/status.rst'.format(os.path.realpath(mkdtemp()))
```

Comment:
I think there is a contextmanager in tmpfile available that allows us to avoid the explicit remove at the end